### PR TITLE
Fix homing in error gcode

### PR DESCRIPTION
**Pull request by elpopo-eng**
When an error occurs in the gcode, we call `PARK`, but following PR #494  we force a home in the `PARK` macro.
I find it strange to do home when the job has an error.
I therefore propose to do `PARK` only when the machine is already homed.

### CODE REVIEW

File: config/machine.cfg [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-      STATUS_LEDS COLOR="ERROR"
-   {% endif %}
-   {% if printer["gcode_macro _USER_VARIABLES"].probe_type_enabled == "dockable" %}
-       _PROBE_ON_ERROR_ACTION
-   {% endif %}
-    CANCEL_PRINT
+    {% if printer["gcode_macro _USER_VARIABLES"].status_leds_enabled %}
+        STATUS_LEDS COLOR="ERROR"
+    {% endif %}
+    {% if printer["gcode_macro _USER_VARIABLES"].probe_type_enabled == "dockable" %}
```
Comment (Frix-x) [inline config/machine.cfg:7]: can you also add a reference to "dockable_virtual" as well ? I think it was forgotten and it's a good idea to add it as well.

**Review by Frix-x — CHANGES_REQUESTED** [review]
I've got a small change to add in this PR to make it more accurate

**Comment by Surion79** [comment]
@elpopo-eng thanks a lot for this PR! this has been introduced with the _CG28 in Park. good it was found early :)

File: config/machine.cfg (hunk 1 above)
Comment (elpopo-eng) [reply config/machine.cfg:7]: Yes ! this make sense.

**Comment by beavis69** [comment]
I've have a head crash at end of print. 
I'm overriding led_effects and the new "done_printing" led effects was not in my override.
So i got an STATUS_LEDS COLOR ERROR followed by a forced homing on my printed parts...

I think this need to be commited to master ASAP

**Comment by Surion79** [comment]
> I've have a head crash at end of print. I'm overriding led_effects and the new "done_printing" led effects was not in my override. So i got an STATUS_LEDS COLOR ERROR followed by a forced homing on my printed parts...
> 
> I think this need to be commited to master ASAP

it was already merged. have you not updated to the latest version?

**Comment by beavis69** [comment]
I think it was merged only in develop branch but not in main, see : https://github.com/Frix-x/klippain/blob/main/config/machine.cfg

**Comment by Frix-x** [comment]
I'll releasea bugfixes version today to sync develop and main

**Pull request merged**